lightning-network-book.tex

```diff
@@ -240,36 +240,36 @@
 \newglossaryentry{clightning}{name={c-lightning},
-  description={Implementation of the Lightning Network Protocol by the Victoria based Blockstream. It is written in C.}}
+  description={Implementation of the Lightning Network Protocol by the Victoria based Blockstream(\url{https://blockstream.com}). Source code is at \url{https://github.com/ElementsProject/lightning}. It is written in C.}}
 \newglossaryentry{shachain}{name={Sha-chain},
   description={}}
 \newglossaryentry{hdwallet}{name={HD Wallet},
   description={}}
 \newglossaryentry{bip}{name={BIP},
-  description={}}
+  description={BIP is short for Bitcoin Improvement Proposals. It is a mechanism by which proposals for the improvement of the Bitcoin network can be made. Submitted proposals are discussed heavily and the best one implemented. See \url{https://github.com/bitcoin/bips}.}}
 \newglossaryentry{bitcoin}{name={Bitcoin},
-  description={}}
+  description={Bitcoin is an experimental digital currency that enables payments to anyone, anywhere in the world. Bitcoin uses peer-to-peer technology to operate with no central authority: managing transactions and issuing money are carried out collectively by the network. See \url{https://www.bitcoin.org} and \url{https://github.com/bitcoin/bitcoin}.}}
 \newglossaryentry{ecash}{name={Electronic Cash},
   description={}}
 \newglossaryentry{ecdsa}{name={ECDSA},
   description={}}
 \newglossaryentry{bech32}{name={bech32},
-  description={}}
+  description={Bech32 is a segwit address format specified by BIP 0173. This address format is also known as "bc1 addresses" because all Bech32 addresses start with "bc" and currently with "bc1". See \url{https://en.bitcoin.it/wiki/Bech32}.}}
 \newglossaryentry{encoding}{name={Encoding},
-  description={}}
+  description={Encoding is the act of translating information from one form to another following specified rules. Text can be encoded as morse signals. Songs can be encoded as MP3 files. The opposite transformation is called "decoding". An MP3 song is decoded as audio.}}
 \newglossaryentry{input}{name={Input Script},
-  description={}}
+  description={The Input Script is a script to prove that you own some bitcoins.}}
 \newglossaryentry{output}{name={Output Script},
-  description={}}
+  description={The Output Script is a script where you designate someone else or multiple other parties to be able to prove they own some bitcoins.}}
 \newglossaryentry{script}{name={Bitcoin Script},
-  description={}}
+  description={Bitcoin uses a scripting system for transactions. It is called "Script" and is simple, stack-based, and processed from left to right. It is intentionally not Turing-complete, with no loops. Scripts can be embedded into Bitcoin transactions.}}
 \newglossaryentry{msat}{name={Millisatoshi},
-description={}}
+description={1000 millisatoshis are one Satoshi. A millisatoshi is hence one hundred billionth of a single bitcoin.}}
```

Comment:
would add that milli satoshi do not exist on the blockchain and are used for rounding purposes in the payment channels of the lightning network.

Reply:
```suggestion
description={1000 millisatoshis are one Satoshi. A millisatoshi is hence one hundred billionth of a single bitcoin. Millisatoshis are used in the Lightning Network only and do not exist on the Bitcoin blockchain. When payments are settled on the Bitcoin blockchain millisatoshis are rounded to the smallest unit on the Bitcoin blockchain, i.e. satoshis.}}
```